core/src/Graph.cpp

```diff
@@ -0,0 +1,102 @@
+#include <gnuradio-4.0/Graph.hpp>
```

Comment:
The GR4 core is and should remain (largely) header-only. Please inline this to the class definition. Also this level of indirection makes this harder to read/refactor and to maintain. Only use `.cpp` impl-files when absolutely needed.